src/framework/uicomponents/qml/MuseScore/UiComponents/Utils.js

```diff
@@ -60,3 +60,12 @@ function getItem(model, row) {
 
     return model[row]
 }
+
+function makeMnemonicText(text){
+    var index = text.indexOf("&")
+    if(index >= 0) {
+        text = text.replace(text.substr(index, 2), ("<u>" + text.substr(index + 1, 1) + "</u>"))
```

Comment:
Technically it can be done with a regex replace, but this is probably more maintainable. Except that I suspect it might crash if the & is somehow at the end of the text.

Reply:
thanks, fixed